src/main/java/br/unb/cic/js/walker/Walker.java

```diff
@@ -139,10 +139,11 @@ public void traverse() {
                 logger.warn("path {} does not exist or isn't a directory", p);
             }
         } catch (IOException ex) {
-            ex.printStackTrace();
+            // ex.printStackTrace();
+            logger.error("failed to execute a concurrent task, reason {}", ex.getMessage());
         } catch (java.lang.InterruptedException | java.util.concurrent.ExecutionException ex) {
             logger.error("failed to execute a concurrent task, reason {}", ex.getMessage());
-            ex.printStackTrace();
+            // ex.printStackTrace();
```

Comment:
The stack trace is important in both cases, leave them as the message may not be the best thing when debugging an exception